packages/moonstone/Button/Button.js

```diff
@@ -42,21 +42,14 @@ const ButtonBase = kind({
 		 * The background-color opacity of this button.
 		 *
 		 * Valid values are:
-		 * * `'opaque'`,
 		 * * `'translucent'`,
 		 * * `'lightTranslucent'`, and
 		 * * `'transparent'`.
 		 *
 		 * @type {String}
-		 * @default 'opaque'
 		 * @public
 		 */
-		backgroundOpacity: PropTypes.oneOf([
-			'opaque',
-			'translucent',
-			'lightTranslucent',
-			'transparent'
-		]),
+		backgroundOpacity: PropTypes.oneOf([null, 'translucent', 'lightTranslucent', 'transparent']),
```

Comment:
The literal `null` can (should) be omitted entirely from this list, since a `null` is implied by simply not including the `backgroundOpacity` property. We're not saying that `<Button backgroundOpacity={null}>` should be used in practice, but instead saying that `<Button>` is a valid option. If this prop were "required", and it needed to support a non-value (for whatever reason), then I'd say we should to keep the `null` literal.

Reply:
I had a bit of confusion with `color` having `null` value as an option. I'll go ahead and make that change as well.

FYI: I don't think there's a good way to include `null` as prop type when `isRequired`. See https://github.com/facebook/react/issues/3163